docs/sdks/android/release-notes.md

```diff
@@ -10,6 +10,36 @@ keywords:
   - android
 ---
 
+## 6.28.0
+
+**Released**: September 10, 2024
+
+### New Features
+
+#### Barcode
+
+* Our [Barode Generator](./barcode-generator/) now allows the generation of Aztec codes. See [`AztecBarcodeGeneratorBuilder`](https://docs.scandit.com/data-capture-sdk/android/barcode-capture/api/barcode-generator-builder.html#class-scandit.datacapture.barcode.generator.AztecBarcodeGeneratorBuilder).
+* On Barcode Capture and SparkScan, we optimized the frame processing time in 4k-UHD resolution with dynamic resolution selection, resulting in up to 35% reduction on high-end devices and consistent performance improvements across all supported devices.
```

Comment:
seems too technical. reduction of 35%? impact to the end user? dynamic resolution might indicates lower robustness? maybe something abstract as "smart"?